RealestateApi/Repository/CustomerRepository.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Npgsql;
+using RealEstateApi.Dto.Response;
+using RealEstateApi.Model;
+using RealEstateApi.Repository.Interfaces; 
+
+namespace RealEstateApi.Repository
+{
+    public class CustomerRepository : ICustomerRepository 
+    {
+        private readonly NpgsqlDataSource _dataSource;
+
+        public CustomerRepository(NpgsqlDataSource dataSource)
+        {
+            _dataSource = dataSource;
+        }
+
+        public async Task<CustomerResponseDto> GetAllCustomersAsync()
+        {
+            CustomerResponseDto customerData = new CustomerResponseDto(); 
+            try
+            {
+                using var conn = await _dataSource.OpenConnectionAsync();
+
+                using var customerQuery = new NpgsqlCommand("SELECT * FROM client;", conn);
+                using var customerReader = await customerQuery.ExecuteReaderAsync();
+
+                while (await customerReader.ReadAsync())
+                {
+                    var customerModel = new ClientModel
+                    {   
+                        Id = customerReader.GetInt32(customerReader.GetOrdinal("id")),
```

Comment:
Since RealEstateRepository and ReferenceDataRepository already have a pattern for this, for instance:

`"Id = (int)realestateReader["id"],"` 

i would like to ask you to improve those 4 lines in order to shadow it's pattern as well